<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditForm</class>
 <widget class="QWidget" name="EditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QSplitter" name="verticalSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>6</number>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="Dense7Pattern">
           <color alpha="255">
            <red>204</red>
            <green>204</green>
            <blue>204</blue>
           </color>
          </brush>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="renderHints">
          <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="requestsTab">
       <attribute name="title">
        <string>Requests</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="UsedRequestsForm" name="requestsForm" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="taskTab">
       <attribute name="title">
        <string>Task settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Select task on diagram</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>User variables</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="debugTab">
       <attribute name="title">
        <string>Debug</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QSplitter" name="debugSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>6</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
              <property name="spacing">
               <number>4</number>
              </property>
              <item>
               <widget class="QToolButton" name="runButton">
                <property name="text">
                 <string>Run</string>
                </property>
                <property name="icon">
                 <iconset resource="../postmonster.qrc">
                  <normaloff>:/icons/debug-run</normaloff>:/icons/debug-run</iconset>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="stepButton">
                <property name="text">
                 <string>Step</string>
                </property>
                <property name="icon">
                 <iconset resource="../postmonster.qrc">
                  <normaloff>:/icons/debug-step</normaloff>:/icons/debug-step</iconset>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="pauseButton">
                <property name="text">
                 <string>Pause</string>
                </property>
                <property name="icon">
                 <iconset resource="../postmonster.qrc">
                  <normaloff>:/icons/debug-pause</normaloff>:/icons/debug-pause</iconset>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="stopButton">
                <property name="text">
                 <string>Stop</string>
                </property>
                <property name="icon">
                 <iconset resource="../postmonster.qrc">
                  <normaloff>:/icons/debug-stop</normaloff>:/icons/debug-stop</iconset>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QStackedWidget" name="progressBarContainer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="progressBarPage" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>2</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QProgressBar" name="taskProgressBar">
                    <property name="value">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="noActiveItemPage">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>2</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>To select item for debugging, double-click on it</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="activeItemPage">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>2</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="activeItemLabel">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QStackedWidget" name="debugStackedWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="environmentPage">
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTreeView" name="environmentTree"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="resultStackedWidget">
              <widget class="QWidget" name="page_2">
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>Select task in tree</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UsedRequestsForm</class>
   <extends>QWidget</extends>
   <header>widgets/usedrequestsform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../postmonster.qrc"/>
 </resources>
 <connections/>
</ui>
